doc/source/getting_started.rst

```diff
@@ -0,0 +1,190 @@
+
+A **RegionAggregationMapping** is a mapping that defines on a per-model
+basis how model native regions should be renamed and aggregated to
+comparison regions.
+
+A **RegionProcessor** is a class that holds a list of
+RegionAggregationMappings and a DataStructureDefinition. This class is
+used to facilitate region processing for model comparison studies.
+
+The structure of a datastructure definition
+-------------------------------------------
+
+A **DataStructureDefinition** is initialized from a folder with the
+following structure:
+
+Variables
+~~~~~~~~~
+
+The *variable* codelist of the **DataStructureDefinition** will be read
+from all yaml files located in a folder of that name (including any
+sub-folders). They must be formatted as a list of dictionaries mapping
+the variable (key) to its attributes.
+
+.. code:: yaml
+
+   - Some Variable:
+       description: A short description
+       unit: A unit
+       <other attribute>: Some text (optional)
+
+Every variable must have a **unit**, which should be compatible with the
+Python package `iam-units <https://github.com/iamconsortium/units>`__.
+The unit attribute can be empty, i.e., the variable is *dimensionless*.
+
+Tags
+^^^^
+
+To avoid repetition (and subsequent errors), any number of yaml files
+can be used as “tags” using a nested list of dictionaries. The files
+defining the tags must have a name starting with ``tag_``.
+
+.. code:: yaml
+
+   - <Tag>:
+     - Some Key:
+         description: a short description of the key
+
+When importing the codelist, any occurrence of ``<Tag>`` in a variable
+name will be replaced by every element in the Tag dictionary. The
+``<Tag>`` will also be replaced in any of the variable attributes.
+
+There must be only one top-level entry in any yaml file to be used as
+tag.
+
+Model Mappings
+--------------
+
+Model mappings, defined in a .yaml format serve three different purposes
+on a per-model basis:
+
+1. Define a list of model native regions that are to be selected (and
+   usually uploaded) from an IAM result. This also serves as an implicit
+   exclusion list for model native regions, since only explicitly
+   mentioned regions are selected.
+
+2. Allow for renaming of model native regions.
+
+3. Define how model native regions should be aggregated to common
+   regions.
+
+This example illustrates how such a model mapping looks like:
+
+.. code:: yaml
+
+   model: model_a
+   native_regions:
+     - region_a: alternative_name_a
+     - region_b
+   common_regions:
+     - common_region_1:
+       - region_a
+       - region_b
+     - common_region_2:
+       - ...
+
+Notes
+~~~~~
+
+-  The names of the three top level keywords ``model``,
+   ``native_regions`` and ``common_regions`` are fixed.
+-  Required properties are ``model`` and **at least** either
+   ``native_regions`` or ``common_regions``. **Both** are **allowed** as
+   well.
+-  **``model``** (str): specifies the model name for which the mapping
```

Comment:
Using both bold and codes doesn't work...

Reply:
Changed to just bold.